nni/nas/benchmarks/nlp/query.py

```diff
@@ -0,0 +1,63 @@
+import functools
+
+from peewee import fn
+from playhouse.shortcuts import model_to_dict
+from .model import NlpTrialStats, NlpTrialConfig
+
+def query_nlp_trial_stats(arch, dataset, reduction=None, include_intermediates=False):
+    """
+    Query trial stats of NLP given conditions.
```

Comment:
NLP -> NLP benchmark

Reply:
need to add more details here, what can be queried with this api?

Reply:
It is answered in this API's docstring's "Returns", but surely I can make more explainations.